core/controllers/dashboard.py

```diff
@@ -261,6 +266,20 @@ def get(self):
         })
 
 
+class ExplorationStats(base.BaseHandler):
```

Comment:
Why is this called ExplorationStats? Maybe ExplorationDashboardDataHandler or something like that.